### PR TITLE
Fixed display of `diverse_with` field in response via APIv2.

### PR DESCRIPTION
**Pull request by dpoltavets**
Closes: #2701

### CODE REVIEW

File: src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/model/FlowDto.java [hunk 1]
```diff
@@ -164,6 +164,9 @@
     @JsonProperty("path_computation_strategy")
     private PathComputationStrategy pathComputationStrategy;
 
+    @JsonProperty("diverse_with")
+    private String[] diverseWith;
```
Comment (timofei-durakov) [inline src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/model/FlowDto.java]: why is it array, not list?

Comment (dpoltavets) [reply src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/model/FlowDto.java]: Changed to Set.

File: src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/common/actions/FlowProcessingAction.java [hunk 2]
```diff
@@ -101,4 +101,11 @@ protected boolean isRemoveCustomerPortSharedCatchRule(String flowId,
 
         return flowIds.size() == 1 && flowIds.iterator().next().equals(flowId);
     }
+
+    protected Set<String> getDiverseWithFlowIds(Flow flow) {
+        return flowRepository.findByGroupId(flow.getGroupId()).stream()
```
Comment (sergii-iakovenko) [inline src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/common/actions/FlowProcessingAction.java]: Does it make sense to fetch a collection of Flow entities and then return just flow ids? Please, consider a new method in the repository.

Comment (dpoltavets) [reply src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/common/actions/FlowProcessingAction.java]: Fixed.

**Pull request merged**